.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,79 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+    This issue template is REQUIRED for all bug reports. It helps us more
+    quickly track, narrow down, and address bugs. 
+
+    Bug reports not adhering to this template will be likely marked invalid and
+    closed as there is very little we can do without the requisite information.
+
+    Thank you for understanding!
+-->
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is.  -->
+
+**Expected Behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Environment**
+<!--
+    This part is incredibly important:
+
+    Please run the following shell command in the OpenRoad root folder:
+
+        ./etc/Env.sh
+
+    Then copy and paste the ENTIRE output between the triple-backticks below. (```)
+
+    If the command does not succeed, you are using an out-of-date version of
+    OpenRoad, and it is recommended that you update.
+-->
+```
+YOUR SURVEY HERE
+```
+
+**To Reproduce**
+<!--
+    You have two options here:
+
+    A. Use `make <SCRIPT_NAME>_issue` to create a tar file with all the files to reproduce the bug(s).
```

Comment:
I think we should add a description for OL as well since many users come to report bugs from that flow.

Reply:
can you provide more context please?

Reply:
@habibayassin I believe Matt is referring to [this](https://openlane.readthedocs.io/en/latest/for_developers/using_or_issue.html?highlight=reproducible). Maybe add a short description of how to run and add a link to the full documentation.

Reply:
I think you can drop `?highlight=reproducible` from the link -- sorry I forgot to remove from the link I sent you. Otherwise LGTM, I will leave to the @maliberty to give the final approval.